api/desecapi/metrics.py

```diff
@@ -46,4 +46,8 @@ def set(name, *args, **kwargs):
 
 #throttling.py metrics
 
-set('desecapi_throttle_failure_occurence', 'number of times throttling failed')
\ No newline at end of file
+set('desecapi_throttle_failure_occurence', 'number of times throttling failed')
+
+#serializers.py metrics
+
+set('desecapi_RRsetSerializer_many_init', 'number of times many_init function is triggerd in RRsetSerializer')
```

Comment:
Nice! That's a very technical description though. Let's use `set('desecapi_rrset_list_serializer', 'number of times RRsetListSerializer was initialized')`